### PR TITLE
Added full test_suite examples with `display_source`

### LINKED ISSUE #2158: Add full test_suite examples with `display_source`
### Summary of Issue

Add full test_suite examples with `display_source` to allow users to see the underlying code examples being imported from test_suite example 

### Issue Type

- [ ] Bug
- [x] Enhancement  
      POEM #___ <N/A or POEM number at https://github.com/OpenMDAO/POEMs>

### Description

N/A

### Example

N/A

### Environment

N/A

### PR DESCRIPTION
**Pull request by DKilkenny**
### Summary

Added full test_suite examples with display_source to allow users to see the underlying code examples being imported from test_suite example

### Related Issues

- Resolves #2158

### Backwards incompatibilities

None

### New Dependencies

None

### CODE REVIEW

File: openmdao/docs/openmdao_book/other_useful_docs/multipoint_beam_opt.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
-
 import numpy as np
 
+om.display_source("openmdao.test_suite.test_examples.beam_optimization.beam_group.BeamGroup")
```
Comment (Kenneth-T-Moore) [inline openmdao/docs/openmdao_book/other_useful_docs/multipoint_beam_opt.py:3]: What's the reason for printing these to stdout when this file is run (or imported from)? I think this is just used for the om command doc.

**Review by Kenneth-T-Moore — CHANGES_REQUESTED** [review]

File: openmdao/docs/openmdao_book/other_useful_docs/multipoint_beam_opt.py (hunk 1 above)
Comment (DKilkenny) [reply openmdao/docs/openmdao_book/other_useful_docs/multipoint_beam_opt.py]: fixed

**Review by Kenneth-T-Moore — APPROVED** [review]

File: openmdao/docs/openmdao_book/features/building_blocks/drivers/doe_driver.ipynb [hunk 2]
```diff
@@ -327,6 +327,11 @@
     "\n",
     "import numpy as np\n",
     "\n",
+    "if rank == 0:\n",
+    "    print(\"Found Case\")\n",
+    "else:\n",
+    "    print(\"Found Case\")\n",
+    "    \n",
```
Comment (swryan) [inline openmdao/docs/openmdao_book/features/building_blocks/drivers/doe_driver.ipynb:334]: this `if` statement seems unnecessary...

Comment (DKilkenny) [reply openmdao/docs/openmdao_book/features/building_blocks/drivers/doe_driver.ipynb]: D'oh! Removed

**Review by swryan — APPROVED** [review]

**Pull request merged**